README.md

```diff
@@ -140,6 +140,19 @@ uv sync --extra all
 
 Tests are run with *pytest*. If you are not familiar with this package, you can find out more on [their website](https://pytest.org/).
 
+The tests rely on some external services running, including postgres, azurite and mongo. Currently, themost reliable way to run tests,
+without setting these services up manually on your machine, is to use the tests devcontainer docker compose configuration.
+
+```shell
+make test
```

Comment:
Does this command also work *inside* the dev container? (I'm starting up a dev container now to see)
If not then I assume the instruction would be simply "uv run pytest" inside the container.